.github/workflows/master-ci.yaml

```diff
@@ -15,14 +15,14 @@ jobs:
           path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
           restore-keys: |
-          ${{ runner.os }}-yarn-
+            ${{ runner.os }}-yarn-
       - name: Use Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: "12.x" # this is the version of citizenfx/node
```

Comment:
FXServer builds released after artifact 4550, should be running Node 16 on the server now.  Although in a build context, it doesn't really matter.